server/modules/index.js

```diff
@@ -2,91 +2,91 @@ module.exports = [
     {
         name: 'util.tokenGenerator',
         dependencies: [],
-        require: __dirname + '/token-generator'
+        module: require(__dirname + '/token-generator')
```

Comment:
Previous approach was 'lazy', i.e. module file was required only upon instantiation. Why do you propose to change this?